sdks/go/pkg/beam/core/util/dot/dot.go

```diff
@@ -24,6 +24,7 @@ import (
 
 	"github.com/apache/beam/sdks/v2/go/pkg/beam/core/graph"
 	"github.com/apache/beam/sdks/v2/go/pkg/beam/internal/errors"
+	"github.com/apache/beam/sdks/v2/go/pkg/beam/model/pipeline_v1"
```

Comment:
Conventionally, we add a short name to Protos. For this package we call it pipepb.